### DISCUSSION TITLE
TextChanged Event

### DISCUSSION THREAD

**Discussion by AviadBly** [AvaloniaUI/Avalonia#12506] [Q&A] [answered]
Hi, I would like to know how to raise a TextChanged event? because I do not have this option for me.
![image](https://github.com/AvaloniaUI/Avalonia/assets/111220386/57b16ff0-149e-4f90-be1a-6aa1bd6091fe)

Also what's the signature of the function should look like? something like this? 

![image](https://github.com/AvaloniaUI/Avalonia/assets/111220386/bd202d4d-6d4f-4314-b8e2-f72b66e71f25)

Thanks

**Comment by timunie** [comment, marked as answer]
in Avalonia you can listen to PropertyChanged on any control. Iirc in 11.0 we have #9157 but not in 0.10.x
